docs/examples/index.maplibre.compare.3857.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8" />
+        <title>Maplibre WGS84 Swipe between raster and vector maps Demo</title>
+        <meta name="viewport" content="initial-scale=1,maximum-scale=1,user-scalable=no" />
+        <script
+            src="https://api.mapbox.com/mapbox-gl-js/v2.3.0/mapbox-gl.js"
```

Comment:
mapbox gl?

Reply:
I have removed this example, it seems there is no unpkg for [maplibre-gl-compare.js](https://github.com/maplibre/maplibre-gl-compare/blob/main/dist/maplibre-gl-compare.js).